onedal/ensemble/forest.py

```diff
@@ -40,6 +44,11 @@
 )
 
 
+def get_hyperparameters_ver(*args):
+    if daal_check_version((2024, "P", 300)):
+        return get_hyperparameters(*args)
```

Comment:
This version check overcomplicates code, while you can just modify original `get_hyperparameters` to return None:
```python
def get_hyperparameters(algorithm, op):
    return hyperparameters_map.get((algorithm, op), None)
```